src/qt/radiobox.cpp

```diff
@@ -149,7 +152,7 @@ bool wxRadioBox::Create(wxWindow *parent,
     // GetMajorDim() is the number of rows.
     if ( style & wxRA_SPECIFY_COLS )
         m_qtBoxLayout = new QHBoxLayout;
-    else if ( style & wxRA_SPECIFY_ROWS )
+    else
```

Comment:
This doesn't seem to be related to the rest of the changes, probably merits its own commit.

Reply:
Is there an easy way to split the commit?  I lumped it together as without the an unconditional else m_qtBoxLayout can be left uninitialised,  but I agree that it isn't consistent with the rest of the commit.

Reply:
Well, it's easy to do with `git rebase --interactive`, but only if you're used to using it (basically you need to "edit" this commit in the interactive rebase TODO list and then separately commit this part, with its own commit message, and the rest). Alternatively, you could amend this commit to undo this change manually and then make a separate commit with it.